src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -2609,8 +2607,8 @@ test_snapshot_query_example_1 (void)
       return;
    }
 
-   if (test_framework_get_server_version () < test_framework_str_to_version ("5.0.0")) {
-      MONGOC_DEBUG ("Skipping test. Test requires server version 5.0.0\n");
+   if (!test_framework_max_wire_version_at_least (MONGOC_READ_SECONDARY)) {
```

Comment:
```suggestion
   if (!test_framework_max_wire_version_at_least (WIRE_VERSION_SNAPSHOT_READS)) {
```
To match the updated message below. (`MONGOC_READ_SECONDARY` is an unrelated macro constant.)

Reply:
Oof 🤦. Well spotted.